packages/ui5-application-inquirer/src/prompts/validators.ts

```diff
@@ -1,7 +1,7 @@
 import { validateModuleName } from '@sap-ux/project-input-validator';
 import { appPathExists } from './prompt-helpers';
 import { t } from '../i18n';
-import { findRootsForPath } from '@sap-ux/project-access';
+import { findCapProjectRoot, findRootsForPath, getCapProjectType, isCapProject } from '@sap-ux/project-access';
```

Comment:
`getCapProjectType` import no longer needed (code smell)

Reply:
Thank you, i have removed it :)